contributed_definitions/nyaml/NXmanipulator.yaml

```diff
@@ -13,36 +13,106 @@ NXmanipulator(NXobject):
+    doc: Amperemeter measuring the drain current of the sample and sample holder.
+    measurement:
+      enumeration: [current]
+    value(NX_FLOAT):
+      unit: NX_TEMPERATURE
+      doc: |
+        In the case of a fixed drain current measurement, this is the scalar drain current measured between
+        the sample and sample holder. It can also be an 1D array of measured currents (without time stamps).
+    value_log(NXlog):
+      value(NX_NUMBER):
+        doc: |
+          In the case of an experiment in which the current changes and is recorded with
+          time stamps, this is an array of length m of currents.
+  sample_bias_potentiostat(NXsensor):
     doc: |
-      Temperature of the cryostat (coldest point)
-  heater_power(NX_FLOAT):
-    unit: NX_POWER
+      Acutator applying a voltage between sample and sample holder.
+    measurement:
+      enumeration: [voltage]
+    setpoint(NX_FLOAT):
+      unit: NX_TEMPERATURE
+      doc: |
+        In the case of a fixed applied bias, this is the scalar voltage applied between
+        sample and sample holder. It can also be an 1D array of voltage setpoints (without time stamps).
+    setpoints_log(NXlog):
+      value(NX_NUMBER):
+        doc: |
+          In the case of an experiment in which the bias is changed and the setpoints are
+          recorded with time stamps, this is an array of length m of voltage setpoints.
+  sample_bias_voltmeter(NXsensor):
     doc: |
-      Power in the heater for temperature control.
-  sample_temperature(NX_FLOAT):
-    unit: NX_TEMPERATURE
+      Sensor measuring the voltage applied to sample and sample holder.
+    measurement:
+      enumeration: [voltage]
+    value(NX_FLOAT):
+      unit: NX_TEMPERATURE
+      doc: |
+        In the case of a fixed applied bias, this is the scalar voltage measured between
+        sample and sample holder. It can also be an 1D array of measured voltages (without time stamps).
+    value_log(NXlog):
+      value(NX_NUMBER):
+        doc: |
+          In the case of an experiment in which the bias changes and is recorded with
+          time stamps, this is an array of length m of voltages.
+  (NXactuator):
     doc: |
-      Temperature at the closest point to the sample. This field may also be found in
-      NXsample if present.
-  drain_current(NX_FLOAT):
-    unit: NX_CURRENT
+      Any additional sensors on the manipulator used to control an external condition. 
+  (NXsensor):
     doc: |
-      Current to neutralize the photoemission current. This field may also be found in
-      NXsample if present.
-  sample_bias(NX_FLOAT):
-    unit: NX_CURRENT
-    doc: |
-      Possible bias of the sample with respect to analyser ground. This field may
-      also be found in NXsample if present.
+      Any additional sensors on the manipulator used to monitor an external condition. 
   (NXpositioner):
     doc: |
       Class to describe the motors that are used in the manipulator
-  (NXfabrication):
```

Comment:
Why was this removed?

Reply:
accidental, fixed. Thanks for spotting it.